public/wp-content/themes/ColoredCow/.eslintrc

```diff
@@ -0,0 +1,3 @@
+{
+	"extends":"rallycoding"
```

Comment:
Yeah got the eslint but what's rallycoding? Lint Plugin? You should consider using one that's written for WP. 

In case you want to consider: https://github.com/WordPress-Coding-Standards/eslint-plugin-wordpress